### ISSUE TITLE
collect network usage

### ISSUE THREAD

**Issue by majst01** [metal-stack/firewall-policy-controller#7]
In order to be able to collect network usage data i propose the following solution:

- install node-exporter on the firewall which exposes:
  - node_network_receive_packets_total{device=vrf123} 12234
  - node_network_transmit_packets_total{device=vrf123} 12234
- firewall-policy-controller will scrape these and store them in a CRD `MetalNetworkStatistics` ?
- kube-counter can then collect them as it does for pods and volumes.

We should add a flag to every network in metal-api the specify if this network must be accounted or not.

@chbmuc @Gerrit91 @mwindower

**Comment by mwennrich** [comment]
just a reminder: network usage gets also collected on the switch interfaces

**Comment by majst01** [comment]
Yes but scraping these metrics from the switch and put them into relation to a actual cluster is difficult.
